edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/DeviceScopeApiClient.cs

```diff
@@ -73,14 +74,14 @@ public Task<ScopeResult> GetIdentity(string targetDeviceId, string targetModuleI
                 .GetOrElse(
                     () =>
                     {
-                        string relativeUri = InScopeIdentitiesUriTemplate.FormatInvariant(this.deviceId, this.moduleId, this.batchSize, null, ApiVersion);
+                        string relativeUri = InScopeIdentitiesUriTemplate.FormatInvariant(this.TargetEdgeDeviceId, this.moduleId, this.batchSize, null, ApiVersion);
                         var uri = new Uri(this.iotHubBaseHttpUri, relativeUri);
                         return uri;
                     });
 
         internal Uri GetServiceUri(string targetDeviceId, string targetModuleId)
         {
-            string relativeUri = InScopeTargetIdentityUriFormat.FormatInvariant(this.deviceId, this.moduleId, targetDeviceId, targetModuleId, ApiVersion);
+            string relativeUri = InScopeTargetIdentityUriFormat.FormatInvariant(this.TargetEdgeDeviceId, this.moduleId, targetDeviceId, targetModuleId, ApiVersion);
```

Comment:
it is a little confusing here using TargetEdgeDeviceId and targetDeviceId, either renamse TargetEdgeDeviceId  to edgeOnBehalfOfDeviceId or similar or targetDeviceId is more like requested identity.